evadb/optimizer/statement_to_opr_converter.py

```diff
@@ -228,9 +230,35 @@ def _visit_union(self, target, all):
         self._plan.append_child(right_child_plan)
 
     def _visit_projection(self, select_columns):
+        def __construct_llm_nodes(llm_exprs):
+            return [LogicalLLM(llm_expr) for llm_expr in llm_exprs]
+
+        llm_exprs, remaining_exprs = extract_llm_expressions_from_project(
+            select_columns
+        )
+        llm_nodes = __construct_llm_nodes(llm_exprs)
+
+        select_columns = []
+        for expr in llm_exprs:
+            select_columns.extend(get_bound_func_expr_outputs_as_tuple_value_expr(expr))
+
+        select_columns.extend(remaining_exprs)
+
+        # add llm plan nodes
+        if llm_nodes:
+            # add existing plan as a child of llm
+            plan_root = LogicalLLM(llm_exprs[0])
+            plan_root.append_child(self._plan)
+            for expr in llm_exprs[1:]:
+                new_root = LogicalLLM(expr)
+                new_root.append_child(plan_root)
+                plan_root = new_root
+            self._plan = plan_root
+
```

Comment:
IMO, the generic way is to do it in optimizer with apply and merge. What will the plan looks like, if we have `SELECT id, LLM(...) FROM some_table;`

Reply:
It will be `Project(id, llm.response) -> LLMExec() -> Get`